app/helpers/service_provider_helper.rb

```diff
@@ -35,6 +35,12 @@ def sp_logo_preview_path(file_name)
       file
   end
 
+  # Generate the list for the SP edit form, including a nil entry
+  def redirect_uri_list(service_provider = @service_provider)
```

Comment:
Since we only call this from one place, can we drop the default argument? Either just use `@service_provider` in the helper or put `@service_provider` directly in the template as an argument?

My thinking is that it makes future refactoring harder if you have to guess